src/widgets/allergies/allergy-card-level-three.test.tsx

```diff
@@ -9,6 +9,10 @@ import { patient, mockAllergyResult } from "../../../__mocks__/allergy.mock";
 const mockGetPatientAllergyByPatientUuid = getPatientAllergyByPatientUuid as jest.Mock;
 const mockUseCurrentPatient = useCurrentPatient as jest.Mock;
 
+jest.mock("react-i18next", () => ({
```

Comment:
@denniskigen :thumbsup: just a suggestion rather than having to write this mock in every file we use translation can this work for our case, @fatmali what are your thoughts 
https://medium.com/@arivu_a/how-to-mock-functions-globally-in-jest-f267fedf7683

Reply:
nice find @donaldkibet , this looks simpler and much more elegant.

Reply:
Even better @donaldkibet. I'll amend the PR.

Reply:
A more concise way of mocking the i8next dependency globally is presented in their repo [here](https://github.com/i18next/react-i18next/tree/master/example/test-jest).